src/test/java/BaseTest.java

```diff
@@ -40,11 +41,16 @@ public static void chromeConfigs() {
 
     @BeforeMethod
     public void launchBrowser() {
-        driver = new ChromeDriver();
+       // driver = new ChromeDriver();
+
+        System.setProperty("webdriver.gecko.driver","geckodriver.exe");
+        driver = new FirefoxDriver();
+        Actions actions = new Actions(driver);
         driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
         driver.manage().window().maximize();
         url = "https://bbb.testpro.io/#!/home";
         driver.get(url);
+
```

Comment:
Please add the method `tearDownBrowser()`

**@AfterMethod** annotated method will be run after each test method 

The **driver.quit()** is used to quit the whole browser session along with all the associated browser windows, tabs and pop-ups.

For example

```
  @AfterMethod
  public void tearDownBrowser() {
    driver.quit();
  }
```